package.json

```diff
@@ -1,5 +1,106 @@
 {
   "dependencies": {
-    "@material-ui/core": "^4.11.2"
+    "@material-ui/core": "^4.11.2",
+    "lit-element": "^2.2.1",
+    "@polymer/polymer": "3.2.0",
+    "@webcomponents/webcomponentsjs": "^2.2.10",
+    "@vaadin/vaadin-grid": "5.7.7",
```

Comment:
We shouldn't have any Vaadin dependencies in `0.2.0`. Did you get this from `main`?

Reply:
I branched off of 0.2.0, so I really don't know why I have those Vaadin dependencies in my branch.

Reply:
I just checked my local 0.2.0 branch to make sure the dependencies didn't contain Vaadin, and that branch was fine (Vaadin-free).  I branched off of that local branch to create my feature branch, so I have no idea how this happened (I probably merged incorrectly when I tried to push to my remote branch).  I am trying to push the correct package.json file up now

Reply:
Ok, the package.json file is fixed now.